### PR TITLE
Note for Up/Download problems on Win WebDAV client

### PR DESCRIPTION
**Pull request by RealRancor**
More fun with the Windows WebDAV client :see_no_evil:

### CODE REVIEW

File: source/dav/clients/windows.md [hunk 1]
```diff
@@ -183,9 +183,13 @@ from a WebDAV volume:
   Vista or XP SP1 with security update 896426 installed)
 * "Error 0x800700DF: The file size exceeds the limit allowed and cannot be
   saved." (Windows 7)
+* "An unexpected error is keeping you from copying the folder. If you continue
+  to receive this error, you can use the error code to search for help with
+  this problem." (Windows 7)
 
 The limitation can only be disabled on client side, by changing registry keys.
-See [this article][3] for detailed instructions.
+See [this article][3] for detailed instructions. Additional informations are
```
Comment (staabm) [inline source/dav/clients/windows.md]: `information`

File: source/dav/clients/windows.md [hunk 2]
```diff
@@ -263,6 +267,14 @@ if you expect an immediate update.
 
 This can be disabled using a registry key. See [http://technet.microsoft.com/en-us/library/ee683963%28v=ws.10%29.aspx](http://technet.microsoft.com/en-us/library/ee683963%28v=ws.10%29.aspx) for more info.
 
+### upload timeouts
+
+An upload of large Files taking longer than 30 minutes using Web Client in
```
Comment (staabm) [inline source/dav/clients/windows.md]: `files`

**Comment by RealRancor** [comment]
@staabm Thanks for the feedback, have fixed them.

**Comment by evert** [comment]
Thank you!

**Comment by Hywan** [comment]
s/fun/drama

**Pull request merged**